client/blocks/author-selector/index.jsx

```diff
@@ -241,9 +242,9 @@ const SwitcherShell = React.createClass( {
 	_onSearch: function( searchTerm ) {
 		this.props.updateSearch( searchTerm );
 	},
-} );
+} ));
 
-module.exports = React.createClass( {
+module.exports = localize(React.createClass( {
```

Comment:
Related: #18710 will soon be ready for review (once a desc is written).
It aims to fix up instances similar to this ^

It currently only looks at `export default`s but could be tweaked to consider `module.exports =` too :)